content/python/concepts/substrings/substrings.md

```diff
@@ -0,0 +1,105 @@
+---
+Title: 'Substrings'
+Description: 'A substring is a sequence of characters that are part of an original string.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+A substring is a sequence of characters that are part of an original string. In Python, substrings can be obtained by using the slicing feature on a [string](https://www.codecademy.com/resources/docs/python/strings) variable. A slice can be made in a specific position within the string or it can be made at the default index.
```

Comment:
```suggestion
A substring is a sequence of characters that are part of an original [string](https://www.codecademy.com/resources/docs/python/strings). In Python, substrings can be obtained by using the slicing feature on a string variable. A slice can be made in a specific position within the string or it can be made at the default index.
```